### PR TITLE
Adding support for input icons with additional properties and render …

### PR DESCRIPTION
**Pull request by ben-schumacher**
…logic.

### CODE REVIEW

File: src/forms/input/index.tsx [hunk 1]
```diff
@@ -47,7 +47,17 @@ export class Input extends React.PureComponent<InputProps> {
...
+        const {
+            className, //prettier
+            disabled,
+            helperText,
+            label,
+            isBoxed,
+            placeholder,
+            size,
+            type,
+            children,
```
Comment (akate1) [inline src/forms/input/index.tsx]: I think we also need to add  prop named as 'children' in InputProps.
For example ,children?: ReactNode | ReactNode[];

Comment (ben-schumacher) [reply src/forms/input/index.tsx]: @akate1, this is not required as `children` is a property always available in React components.

Comment (akate1) [reply src/forms/input/index.tsx]: ok got it.

**Comment by girishdudhe** [comment]
Why the name of file is index.tsx? It should be Input.tsx
Is there any specific reason?

**Comment by ben-schumacher** [comment]
> Why the name of file is index.tsx? It should be Input.tsx
> Is there any specific reason?

I agree, renamed.  Thanks @girishdudhe !

**Pull request merged**